cirq-core/cirq/study/resolver.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, param_dict: 'cirq.ParamResolverOrSimilarType' = None) -> None
 
     def value_of(
         self, value: Union['cirq.TParamKey', float], recursive: bool = True
-    ) -> 'cirq.TParamVal':
+    ) -> Union['cirq.TParamVal']:
```

Comment:
don't understand this one.  Union with what?

Reply:
Reverted.